config/locales/es/es.yml

```diff
@@ -389,6 +389,7 @@ es:
 
   project_group:
     add: "Translate Me"
+    edit: "Translate Me"
```

Comment:
Can you please add a happy path feature spec, one for adding child and one for editing? I think we need to start doing these as part of frontend issues.

I suggest adding `  gem "capybara-screenshot", "~> 1.0"` to the gemfile. It helps a lot for debugging.

We will also need a web driver. Do you use poltergeist? If so please add it or suggest an alternative. Also can you please add PhantomJS or equivalent to the Readme.md as a dependency for dev envs.

Thank you!